services/ui-src/src/components/templateCards/TemplateCard.test.tsx

```diff
@@ -16,23 +18,24 @@ jest.mock("@chakra-ui/transition", () => ({
   )),
 }));
 
+const verbiage = templateCardsVerbiage.MCPAR;
 const templateCardComponent = (
   <RouterWrappedComponent>
-    <TemplateCard templateName="MCPAR" />
+    <TemplateCard verbiage={verbiage} />
   </RouterWrappedComponent>
 );
 
-describe("Test Template Download Card Item", () => {
+describe("Test TemplateCard", () => {
```

Comment:
Changed these to use the component's name, because then you can run just this test file with the `-t` flag, like `yarn test -t TemplateCard`.